README.md

```diff
@@ -274,7 +331,7 @@ try (OakMap sub = oak.subMap(from, false, to, true)) {
 ##### Direct ByteBuffer Transformations
 
 ```java
-Function<ByteBuffer, String> toStrings = e -> String.valueOf(e.getInt(0));
+Function<OakReadBuffer, String> toStrings = e -> String.valueOf(e.getInt(0));
 
 Iterator<String> iter = oak.zc().values().stream().map(v -> v.transform(toStrings)).iterator();
 while (iter.hasNext()) {
```

Comment:
1) There need to be OakUnsafeDirectBuffer usage example.
2) Stream iterators need to be added... although not part of this effort. If you prefer not to do it here please at least open an issue.